packages/salesforcedx-vscode-visualforce/package.json

```diff
@@ -68,7 +68,11 @@
       {
         "language": "visualforce",
         "scopeName": "text.visualforce.markup",
-        "path": "./syntaxes/visualforce.json"
+        "path": "./syntaxes/visualforce.json",
+        "embeddedLanguages": {
```

Comment:
See https://github.com/Microsoft/vscode/blob/44730064979184eed5ace0188e7e391d60febbb1/extensions/html/package.json#L62

We only support CSS and JavaScript.